File: to-do app/to-do-java/src/main/java/com/renan/todo/business/UserService.java
package com.renan.todo.business;

/**
 * Services for user info update and password update
 */
public interface UserService {
    
    /**
     * Update information about the user
     *
     * @param id       - user id
     * @param name     - user first name
     * @param lastName - last name
     * @throws BusinessException - in cases of any errors or invalid operations
     */
    public void updateUser(Integer id, String name, String lastName) throws BusinessException;

    /**
     * Update information about the user
     *
     * @param id       - user id
     * @param password - new password
     * @throws BusinessException - in cases of any errors or invalid operations
     */
    public void updateUserPassword(Integer id, String password) throws BusinessException;

}

File: to-do-app/to-do-kotlin/src/main/kotlin/com/renan/todo/service/UserService.kt
package com.renan.todo.service

/**
 * Services for user info update and password update
 */
interface UserService {

    /**
     * Update information about the user
     */
    @Throws(BusinessException::class)
    fun updateUser(id: Int?, name: String?, lastName: String?)

    /**
     * Update information about the user
     */
    @Throws(BusinessException::class)
    fun updateUserPassword(id: Int?, password: String?)

}